model/task_lifecycle.go

```diff
@@ -1198,8 +1201,74 @@ func UpdatePatchStatus(p *patch.Patch, versionStatus string) error {
 		if err = p.UpdateStatus(patchStatus); err != nil {
 			return errors.Wrapf(err, "updating patch '%s' with status '%s'", p.Id.Hex(), patchStatus)
 		}
+		if p.IsGithubPRPatch() {
+			if err = SendVersionStatusToGithub(p.Id.Hex(), p.GithubPatchData.BaseOwner, p.GithubPatchData.BaseRepo, p.GithubPatchData.HeadHash, "patch status change", "pr-task-reset"); err != nil {
```

Comment:
I didn't look into this too much but I'm wondering if instead of this function we could repurpose makeGithubStatusUpdateJob 
https://github.com/evergreen-ci/evergreen/blob/e37538a7fd96246c2241148634adf2b4beabc438/units/github_status_api.go#L85

Reply:
Alas units cannot be imported into the model package, that is what my initial solution was.  It also seems like running amboy jobs from the model package is a pattern we avoid.  Although I'm not necessarily opposed to refactoring the packages a bit to make it possible.